misc/afl-libfuzzer/afl-libfuzzer-daemon.py

```diff
@@ -621,6 +635,39 @@ def test_binary_asan(bin_path):
     return False
 
 
+def apply_transform(script_path, testcase_path):
+    """
+    Apply a post-crash transformation to the testcase
+
+    @type script_path: String
+    @param script_path: Path to the transformation script
+
+    @type testcase_path: String
+    @param testcase_path: Path to the testcase
+
+    @rtype: String
+    @return: Path to the archive containing the original and transformed testcase
+    """
+
+    with tempfile.TemporaryDirectory() as output_path:
+        try:
+            subprocess.check_call([script_path, testcase_path, output_path])
+        except subprocess.CalledProcessError:
+            raise Exception("Failed to apply post crash transformation.  Aborting...")
+
+        if len(os.listdir(output_path)) == 0:
+            raise Exception("Transformation script did not generate any files.  Aborting...")
+
+        archive_path = testcase_path + ".zip"
+        with ZipFile(archive_path, 'w') as archive:
+            archive.write(testcase_path)
+            for root, dirs, files in os.walk(output_path):
+                for file in files:
+                    archive.write(os.path.join(root, file))
```

Comment:
I'm not sure this is correct. With `output_path` being a full path (`/tmp/something`), that will the full path structure into the zip, won't it? Can you double check that this just includes one relative directory and in general the zip looks like you would expect it to?

Reply:
You are correct.  I've updated this to use pathlib and only include directories relative to the output path.